src/psyclone/tests/psyir/transformations/parallel_loop_trans_test.py

```diff
@@ -494,9 +494,6 @@ def test_paralooptrans_validate_all_vars(fortran_reader):
     with pytest.raises(TransformationError) as err:
         trans.validate(loop)
     err_text = str(err.value)
-    # Should have two messages (the first being the one that is ignored by
-    # validate).
-    assert "'ipivot' is only written once" in err_text
```

Comment:
How come this warning has gone?

Reply:
Before it was printing all messages reported by the DA (irrespective that these were not parallelisation issues). Now we just report all the issues that cause the validate to fail. I think this is better for the user to "know what it has to be fixed".

Reply:
Oh nice.